include/functions.h

```diff
@@ -1181,11 +1181,11 @@ void Sample_Init(GameState* thisx);
 void Inventory_ChangeEquipment(s16 equipment, u16 value);
 u8 Inventory_DeleteEquipment(PlayState* play, s16 equipment);
 void Inventory_ChangeUpgrade(s16 upgrade, s16 value);
-s32 Object_Spawn(ObjectContext* objectCtx, s16 objectId);
-void Object_InitBank(PlayState* play, ObjectContext* objectCtx);
-void Object_UpdateBank(ObjectContext* objectCtx);
-s32 Object_GetIndex(ObjectContext* objectCtx, s16 objectId);
-s32 Object_IsLoaded(ObjectContext* objectCtx, s32 bankIndex);
+s32 Object_SpawnPersistent(ObjectContext* objectCtx, s16 objectId);
+void Object_InitContext(PlayState* play, ObjectContext* objectCtx);
+void Object_UpdateLoadEntries(ObjectContext* objectCtx);
+s32 Object_GetLoadEntryIndex(ObjectContext* objectCtx, s16 objectId);
+s32 Object_IsLoadEntryLoaded(ObjectContext* objectCtx, s32 loadEntryIndex);
```

Comment:
I agree we can't really abstract the loading part away, but I don't think we need to have "load" in every name either, especially since anyone reading a bit of the code will quickly see all the `Object_IsLoaded`-type calls stuff and figure out it's important